Epic/hackaton q2 2025 kyb and individuals pages

chesterkmr opened the pull request:
- Added Identity verification Page & Table & Creation dialog
- Added kyb & ubos check Page & Table Creation dialog

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **New Features**
  - Introduced KYB (Know Your Business) & Ownership and Identity Verification checks pages, including creation dialogs, advanced filtering, and detailed check views.
  - Added interactive tables for KYB/UBO and Identity Verification checks with sorting, status badges, pagination, and no-data states.
  - Implemented status management with badges, dropdowns, and confirmation dialogs for both KYB/UBO and Identity Verification checks.
  - Added section observer navigation for improved page navigation within detailed check views.
  - Enhanced localization for new toast notifications and UI strings.

- **Bug Fixes**
  - Improved error handling and validation for check creation and status updates.

- **Chores**
  - Updated backend and API to support new checks endpoints, DTOs, and service modules.
  - Extended customer configuration to control feature availability for KYB/UBO and Identity Verification checks.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

chesterkmr committed:
feat: added identity verficaition page boilerplate

chesterkmr committed:
feat: implemented create kyb & ubo check form

chesterkmr committed:
feat: added mocks for kyb & ubos page

chesterkmr committed:
feat: implemented mocked identity verification & fixes

chesterkmr committed:
feat: updated kyb & stakeholders design

chesterkmr committed:
feat: added initial checks module

chesterkmr committed:
feat: fixed checks endpoint

chesterkmr committed:
feat: connected backoffice to kyb & ubo endpoint

chesterkmr committed:
feat: added check creation requests & updated toast

r4zendev committed:
feat: kyb check page

chesterkmr committed:
feat: added feature flags

chesterkmr committed:
feat: updated schemas & added swagger to checks controller

chesterkmr committed:
fix: resolved imports & fixed structure

jonathan-shemer commented on apps/backoffice-v2/src/pages/KybAndUboCheckPage/KybAndUboCheckPage.tsx:
```diff
@@ -1,8 +1,140 @@
+import { NotesButton } from '@/domains/notes/NotesButton';
+import { MerchantMonitoringReportStatus } from '../MerchantMonitoring/components/MerchantMonitoringReportStatus/MerchantMonitoringReportStatus';
+
+export const KybAndUboCheckPage = () => {
```
following our discussion, let's make sure to re-use existing components from the full onboarding flow here, rather than reimplement those

chesterkmr committed:
fix: finalized table data fetching and mapping

chesterkmr committed:
fix: fixed dtos

chesterkmr committed:
feat: added rendering of merchant id

chesterkmr committed:
chore: disabled findings

chesterkmr committed:
fix: replaced internal with external controller

chesterkmr committed:
fix: reverted main

chesterkmr committed:
fix: fixed fetchers urls

chesterkmr committed:
fix: fixed format

chesterkmr committed:
feat: state is now required only for US

chesterkmr committed:
fix: fixed kyb checks schema

r4zendev committed:
use blocks

chesterkmr committed:
fix: schema fix

r4zendev committed:
upd

chesterkmr committed:
fix: fixed table rendering

tomer-shvadron committed:
refactor(kyb-ubo): improve column definitions and badge rendering

- Simplify column accessor definitions in KybAndUboChecksTable
- Adjust badge rendering to safely access status data properties
- Remove unnecessary console log in KycAndUboLogic hook

r4zendev committed:
upd schema, add data not available placeholders

r4zendev committed:
upd references to check

r4zendev committed:
remove mock

chesterkmr committed:
fix: fixed failed status crash & added polling

chesterkmr committed:
fix: added state to checks payload

r4zendev committed:
remove garbage

r4zendev committed:
fixes

r4zendev committed:
upd

r4zendev committed:
hackathon text fixes

r4zendev committed:
hackathon fixes 2

r4zendev committed:
fix links

r4zendev committed:
feat: added kyb & ubos boilerplate

r4zendev committed:
remove individuals page from router

r4zendev committed:
fixes
